jobs.groovy.j2

```diff
@@ -0,0 +1,433 @@
+{% macro folder(folder, display_name)                                       -%}
+{# creates a folder structure to group jobs
+   param folder:  '<clusterhq-flocker/branchname>'
+   param displayName: Name displayed in the jenkins view for this folder     #}
+println("creating {{folder}}...")
+folder("{{folder}}") { displayName("{{display_name}}") }
+{%- endmacro                                                                -%}
+{#
+
+#}
```

Comment:
Thanks. I think that helps in that it gives someone a place to start. As a first-time user, I kind of want everything explained in comments, but I'm aware that after some experience, those "what this is doing" comments will be in the way.  Given that, I'm inclined to accept the token effort.